C-Plus-Plus/other/distinct_subsets_of_set.cpp

```diff
@@ -0,0 +1,74 @@
+//Finding all distinct subsets of a given set using BitMasking Approach
```

Comment:
Change all single-line comment in one multiline comment.

Reply:
@iamrajiv I have made the suggested changes. Please review the same. Thankyou!